httpserver-rs/src/settings.rs

```diff
@@ -4,6 +4,7 @@
 //! which determine how the configuration is parsed.
 //!
 //! For the key...
+use std::net::{IpAddr, Ipv4Addr};
```

Comment:
should this be with the other imports?

Reply:
Std imports are separated by a line before external crates import, followed by another line before internal imports